ddtrace/bootstrap/sitecustomize.py

```diff
@@ -10,14 +10,15 @@
 
 from ddtrace.utils.formats import asbool, get_env
 from ddtrace.internal.logger import get_logger
-from ddtrace import constants
+from ddtrace import config, constants
 
-logs_injection = asbool(get_env("logs", "injection"))
 DD_LOG_FORMAT = "%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] {}- %(message)s".format(
-    "[dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] " if logs_injection else ""
+    "[dd.version=%(dd.version)s dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] "
```

Comment:
IIUC this is a version from the application, is it worth printing it here?

Reply:
What do you mean?

`DD_VERSION` is meant to represent the version of user's application that is running.

Reply:
Do we need to print the version of the application each time a logline is printed?
I'd think you have it printed e.g. at startup and then it never changes, so printing it every time does not make any sense.